.gitmodules

```diff
@@ -1,4 +1,4 @@
 [submodule "lib/rpc_spec"]
 	path = lib/rpc_spec
-	url = git@github.com:vladmu/rpc_spec.git
+	url = https://github.com/smartdevicelink/rpc_spec.git
```

Comment:
@crokita you pointed the submodule to the correct repository, but please take into account the `master` branch of `rpc_spec` doesn't contain the parser code yet. The corresponding [PR](https://github.com/smartdevicelink/rpc_spec/pull/202) is not merged yet. It would be good if you notify @JackLivio in answer to [this comment](https://github.com/smartdevicelink/rpc_spec/pull/202#issuecomment-577389761) that JS generator passed your checks and `rpc_spec` could be merged too.